ui-participant/src/Navbar.tsx

```diff
@@ -310,11 +310,11 @@ export const AccountOptionsDropdown = () => {
             {user.username}
           </p>
           <hr className="dropdown-divider d-none d-lg-block"/>
-          <NavLink to="/hub/profile">
+          {relations.length === 0 && <NavLink to="/hub/profile">
```

Comment:
```suggestion
          {!relations.length && <NavLink to="/hub/profile">
```

Reply:
Let's put a link to the manage profiles page here too if there are > 0 relations.  I think it's nice to know that the profile editing capability is at least always accessible here, and then for proxy studies it's also accessible from the dashboard selector